api/tax_calculator/controllers.py

```diff
@@ -1,19 +1,16 @@
-import random
-from api.utils import naptime
-from .tax_brackets import get_tax_brackets
+from api import utils
+from api.tax_calculator import tax_brackets
```

Comment:
The reason for the prior import was to prevent stuttering. 
```
tax_brackets.get_all_tax_brackets()
```
tax_brackets is mentioned twice, which is too many times.

Reply:
👀 hmm good point